src/components/AnonymousChat/child/ChatItemMyTextV2.tsx

```diff
@@ -115,55 +40,115 @@ const ChatItemMyTextV2 = ({
+    {title: 'Copy Message', systemIcon: 'square.on.square'},
+    {title: 'Delete Message', systemIcon: 'trash', destructive: true}
+  ];
+
   const renderAvatar = React.useCallback(() => {
-    if (isContinuous) return <View style={styles.avatar} />;
+    if (!isShowUserInfo) return <View style={styles.avatar} />;
     return <View style={styles.ml8}>{avatar}</View>;
   }, []);
 
-  const handleTextContainerStyle = () => {
-    if (chatType === SIGNED) {
-      return [styles.containerSigned, styles.textContainer];
-    }
-    return [styles.containerAnon, styles.textContainer];
+  const onSwipeToReply = (direction) => {
+    if (direction === 'right') return;
+    if (swipeableRef.current) swipeableRef.current?.close();
+    setReplyPreview({
+      username,
+      time,
+      message,
+      messageId: data?.id,
+      chatType,
+      messageType: 'regular'
+    });
   };
 
   return (
-    <View style={styles.chatContainer}>
-      <View style={handleTextContainerStyle()}>
-        {!isContinuous && (
-          <View style={styles.chatTitleContainer}>
-            <Text style={styles.userText}>{username}</Text>
-            <View style={styles.dot} />
-            <Text style={styles.timeText}>{time}</Text>
-          </View>
-        )}
-        <Text ref={messageRef} style={styles.text}>
-          {`${message}`}
-        </Text>
-
-        {renderIcon()}
+    <Swipeable
+      testID="swipeable"
+      ref={swipeableRef}
+      friction={3}
+      overshootFriction={2}
+      onSwipeableOpen={onSwipeToReply}
+      renderLeftActions={replyIcon}>
+      <View style={containerStyle(true, isReplyPrompt)}>
+        <Animated.View style={[styles.wrapper, animatedBubbleStyle]}>
+          <ContextMenu
+            previewBackgroundColor="transparent"
+            style={{flex: 1}}
+            actions={contextMenuActions}
+            onPress={(e) => onContextMenuPressed(e, data, chatType)}>
+            <View style={styles.radius8}>
+              <View style={textContainerStyle(true, chatType, isNewLine)}>
+                {isShowUserInfo && (
+                  <View style={styles.chatTitleContainer}>
+                    <Text style={[styles.userText, textStyle(true)]}>{username}</Text>
+                    <View style={dotStyle(true)} />
+                    <Text style={[styles.timeText, textStyle(true)]}>{time}</Text>
+                  </View>
+                )}
+
+                <ChatReplyView
+                  type={chatType}
+                  messageType={messageType}
+                  replyData={data?.reply_data}
+                />
+
+                <Text style={messageStyle(true, messageType)} onTextLayout={onTextLayout}>
+                  {`${message}`}
+                </Text>
+
+                {renderIcon()}
+              </View>
+            </View>
+          </ContextMenu>
+
+          {renderAvatar()}
+        </Animated.View>
       </View>
-      {renderAvatar()}
-    </View>
+    </Swipeable>
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

In the `Swipeable` component, the `onSwipeableOpen` prop is used to handle the swipe action. However, it's not clear what happens when the user swipes in the opposite direction. Consider adding an `onSwipeableClose` prop to handle this case.

```diff
<Swipeable
  testID="swipeable"
  ref={swipeableRef}
  friction={3}
  overshootFriction={2}
  onSwipeableOpen={onSwipeToReply}
+ onSwipeableClose={onSwipeToCancel}
  renderLeftActions={replyIcon}>
```

In the `ContextMenu` component, the `onPress` prop is used to handle the context menu action. However, it's not clear what happens when the user cancels the context menu. Consider adding an `onCancel` prop to handle this case.

```diff
<ContextMenu
  previewBackgroundColor="transparent"
  style={{flex: 1}}
  actions={contextMenuActions}
  onPress={(e) => onContextMenuPressed(e, data, chatType)}
+ onCancel={onContextMenuCancel}>
```


<!-- This is an auto-generated comment by OSS CodeRabbit -->